INSTALLATION.md

```diff
@@ -82,3 +82,47 @@ Bedrock documentation is available at [https://roots.io/bedrock/docs/](https://r
 
 [roots-wp-salt]:https://roots.io/salts.html
 [wp-cli-dotenv]:https://github.com/aaemnnosttv/wp-cli-dotenv-command
+
+
+
+## Recent alterative with Local
+
+1. Create a site `sample1` with Local inside your regular projects folder (named as `$PATH` below in the commands)
+You will endup with a folder that contains 
+- app 
+- conf 
+- logs
+
+2. Open the terminal and run the commands
+$ `cd $PATH/sample1/app`
+$ `composer create-project dekode/project-base sample1`
+
+At this point you should have a structure like:
+- app 
+	- public
+	- sample1
+- conf 
+- logs
+
+If at this step you get any errors, check the constrains, 
```

Comment:
could you update the current constraints to something that is working?

Reply:
What do you mean? Is that not something visible in the video?

Reply:
No just update the composer.json in this repo to something valid

Reply:
This will break anyway for other dependencies too. Maybe you can change it, I don't want to touch this project code for now.